hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml

```diff
@@ -0,0 +1,55 @@
+---
+nm::connections:
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.81/26,139.229.175.126"
```

Comment:
The device association appears to be missing.
![image](https://github.com/lsst-it/lsst-control/assets/948901/b96abad8-3213-48da-9f41-ba36b6fabab1)

Reply:
Device is registered on ipam.cp.lsst.org.